### DISCUSSION TITLE
How to display offline pack on mapview?

### DISCUSSION THREAD

**Discussion by KONGWaiKit** [rnmapbox/maps#1773] [Q&A]
Hello, I'm working on a project that allows users to use the map offline.

Here's my code to download the offline pack.
![DownloadCode](https://user-images.githubusercontent.com/72793143/157208398-cfea9135-1bc3-4dbf-87f4-59b91d03a787.JPG)

And it returns success.
`{"_metadata": null, "pack": {"bounds": [[Array], [Array]], "metadata": "{\"name\":\"LionRock_HongKong\"}"}} {"completedResourceCount": 680, "completedResourceSize": 11077768, "completedTileCount": 153, "completedTileSize": 4160477, "name": "LionRock_HongKong", "percentage": 100, "requiredResourceCount": 680, "state": 2}` 

Then, I tried to get my offline packs back and make use in MapView
![Code](https://user-images.githubusercontent.com/72793143/157209854-abd9910a-c054-4350-b466-2c8c92bd328d.JPG)
 
When I run the app and cut off the internet access, Mapbox just returns blank.
![Capture](https://user-images.githubusercontent.com/72793143/157210665-6f2044f7-e4c4-4cd2-92d7-e7a8344f3207.JPG)

How can I fix it? Thanks.

**Comment by monuv10** [comment]
Any fix you got??

**Comment by mfazekas** [comment]
I don't think we have a `offlinePacks` attribute, just use the same styleUrl as you're creating the offline pack with.

**Reply by monuv10** [reply]
Can u share example for react native
I am getting "unable to resolve host api.mapbox.com no address associated with hostname" when i try to check map in offline mode

i ma getting pack as
{"pack":{"metadata":"{\"name\":\"Mumbai\"}","bounds":[[72.91134400801224,19.137649377790733],[72.84392790360886,19.014469033228885]]},"_metadata":{"name":"Mumbai"}}

and passing lat long to camera component

 <MapView style={{ flex: 1 }}>
        <Camera zoomLevel={10} centerCoordinate={latLong} />
    </MapView>

is there anything am i missing??

![image](https://user-images.githubusercontent.com/92204264/220368640-73e9e92d-3702-43dd-8761-eb5a90ed3564.png)

Awaiting for you reply

**Reply by TiagoDanin** [reply]
I have the same problem
And the repository example didn't work

**Reply by somasekharkakarla** [reply]
@mfazekas , please provide an example. it so confusing. i am not understanding how using created pack in offline

**Reply by losmi** [reply]
@mfazekas did you find some solution? can you provide an example with code? thnx

**Reply by NitayMG** [reply]
Facing same issue as you guys....any help for expo react native with the new libary of mapbox '@rnmapbox/maps'

**Reply by DBcoppr** [reply]
bound should be -- [[neLng, neLat], [swLng, swLat]]

**Comment by ombogdan** [comment]
any help?

**Comment by djangoamidala** [comment]
you just have to use the same style than you have downloaded. perfectly working for me
